.github/workflows/docker-publish.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - main
+      - 2u/project-theseus
```

Comment:
Do we want to publish to Dockerhub for `2u/project-theseus`?

Asking because I thought no, but want to make sure I didn't miss a reason. From my recollections, we publish to Dockerhub to deploy to stage/prod for our k8s setup, and not sure we want to do that yet.

Reply:
Removed, wasnt sure if it was necessary so I had added it there.